File: aragog-common/src/main/java/com/gongshw/aragog/common/model/ListPageParseResult.java
/*
 * Copyright (C) 2016 Baidu, Inc. All Rights Reserved.
 */
package com.gongshw.aragog.common.model;

import java.util.List;

import lombok.Data;

/**
 * @author gongshiwei
 */
@Data
public class ListPageParseResult {
    private List<DetailPageUrl> detailPageUrls;
    private List<ListPageUrl> listPageUrls;
}

File: aragog-common/src/main/kotlin/com/gongshw/aragog/common/model/ListPageParseResult.kt
package com.gongshw.aragog.common.model;

import java.io.Serializable

/**
 * @author gongshw
 */
data class ListPageParseResult(
        val detailPageUrls: List<DetailPageUrl>,
        val listPageUrls: List<ListPageUrl>
) : Serializable
